test/src/unit-capi-dense_array.cc

```diff
@@ -1266,10 +1266,11 @@ void DenseArrayFx::check_simultaneous_writes(const std::string& path) {
 
   // Pre-generate buffers to write
   for (int i = 0; i < nthreads; i++) {
-    subarrays.push_back({{domain_0_lo,
-                          domain_0_lo + tile_extent_0 - 1,
-                          domain_1_lo,
-                          domain_1_lo + tile_extent_1 - 1}});
+    subarrays.push_back(
```

Comment:
I'm curious what triggered these formatting changes?

Reply:
My editor was defaulting to a different version of clang-format. I'll undo those.